README.md

```diff
@@ -43,6 +43,7 @@ rebuild -u obs_decam obs_cfht validation_data_decam validation_data_cfht validat
 ------
 To setup for a run with CFHT:
 ```
+setup pipe_tasks
```

Comment:
This is unnecessary because validate_drp sets up pipe_tasks. I suggest either updating the table file as you have or updating the ReadMe, but not both.